rust/rope/src/rope.rs

```diff
@@ -730,6 +730,16 @@ impl<'a> From<&'a Rope> for String {
     }
 }
 
+impl fmt::Display for Rope {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut ci = self.iter_chunks_all();
```

Comment:
The function `iter_chunks_all` does not exist. Use `iter_chunks(0..self.len())` instead to get a iterator over the whole rope.

Reply:
Yea, iter_chunks_all was removed a few days ago, the replacement is `iter_chunks(..)`. (we can now specify ranges in these functions, which is convenient).

Reply:
I thought I had my fork synced with remote but it wasnt. I have made changes in my latest commit.